client/src/pages/selection.tsx

```diff
@@ -106,7 +126,7 @@ export default function Selection() {
       <Grid container>
         <Grid item xs={12} md={6}>
           <Box className={classNames(classes.container)}>
-            <h2>Select Payers</h2>
+            <h2>Select Players</h2>
```

Comment:
lol. thank you!!!!!!

Reply:
Any chance since you're here that it should or could be changed to Musicians?